src/index.css

```diff
@@ -5,4 +5,12 @@
 @font-face {
     font-family: 'Neucha';
     src: url('./assets/fonts/Neucha-Regular.ttf');
+}
+
+.container {
```

Comment:
Actually it is recommended to use [this](https://tailwindcss.com/docs/reusing-styles#extracting-classes-with-apply) to apply class in TailwindCSS. So, it would be nice if you could edit this for me :D. Thanks Kub